### DISCUSSION TITLE
Dependabot command for pull request grouping

### DISCUSSION THREAD

**Discussion by BelfordZ** [community/community#45991] [Code Security]
### Select Topic Area

Product Feedback

### Body

Often times I have 5 or 6 dependabot PRs stacked up. Its really annoying to wait for each one to rebase when merging them sequentially.

This issue is to support a command for dependabot that will allow me to tell dependabot to reopen a PR combining any other PR's to the repo that dependabot has opened.

an example command interface might be:

`@dependabot combine https://github.com/org/repo/pull/42069`

Thanks for your time

**Comment by jge162** [comment]
sorry but there isn't currently a feature in Dependabot to combine multiple pull requests into one. On going issue. -> https://github.com/testcontainers/testcontainers-java/issues/4172
